common.blocks/i-bem/i-bem.bemhtml

```diff
@@ -250,11 +250,41 @@ BEMContext.prototype.generateId = function generateId() {
 var oldApply = exports.apply;
 exports.apply = BEMContext.apply = function BEMContext_apply(context) {
     var ctx = new BEMContext(context || this, oldApply);
+
     ctx.apply();
+
     return ctx._str;
 };
 
-BEMContext.prototype.reapply = BEMContext.apply;
+BEMContext.prototype.reapply = function reapply() {
+    var mode = typeof $$mode !== 'undefined' && $$mode,
```

Comment:
@zxqfox this was introduced for 20% speed boost. `BEMContext` is way too polymorphic, and access these properties was very slow.